### PR TITLE
[FR] Skinning

### PR DESCRIPTION
**Pull request by DoryanR**
Create French translation of osu! wiki - Skinning Page

### CODE REVIEW

File: wiki/Skinning/fr.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Skinning
+
+Le skinning est l'une des principales caractéristiques d'osu!. Il permet aux joueurs de dériver des éléments de skinning originaux pour créer les leurs! Les skins peuvent varier d’être amusants, thématiques, PRO ou presque impossibles à jouer.
```
Comment (UltayJr) [inline wiki/Skinning/fr.md]:
```suggestion
Le skinning est l'une des principales caractéristiques d'osu!. Il permet aux joueurs de modifier les éléments de skinning originaux pour créer les leurs ! Les skins peuvent varier, ils peuvent être amusant, ou pour des skins de pro, avec des thèmes mais aussi presque impossibles à jouer.
```

File: wiki/Skinning/fr.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Comment puis-je faire un skin
+
+*Page principale: [Tutoriel de skinning](/wiki/Skinning/Tutorial)*
+
+## Listes d'éléments de skins
+
+La liste des éléments de skinning est divisée en ces catégories:
+
+- [Skinning Interface](/wiki/Skinning/Interface)
```
Comment (UltayJr) [inline wiki/Skinning/fr.md]:
```suggestion
- [Skinning d'interface](/wiki/Skinning/Interface)
```

File: wiki/Skinning/fr.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Skinning
+
+Le skinning est l'une des principales caractéristiques d'osu!. Il permet aux joueurs de modifier les des éléments de skinning originaux pour créer les leurs! Les skins peuvent varier, ils peuvent amusant, ou pour des skins de pro, avec des thèmes mais aussi.
```
Comment (Pachiru) [inline wiki/Skinning/fr.md]:
```suggestion
Le skinning est l'une des principales caractéristiques d'osu!. Cela permet aux joueurs de modifier les éléments de skinning originaux pour créer les leurs! Les skins peuvent varier, ils peuvent être amusant, thématisés ou alors être plus professionnel et sobre.
```

File: wiki/Skinning/fr.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+## Skinning sets
+
+*Page principale: [Liste de skins](/wiki/Ranking_Criteria/Skin_Set_List)*
+
+**Pour les beatmaps seulement,** Si votre skin beatmap contient un seul élément parmi les ensembles répertoriés dans les critères de classement, il doit contenir tous les autres éléments dudit ensemble. Cela ne s'applique qu'aux skins beatmap, mais d'autres skinners voudront peut-être aussi prendre en compte ces listes.
```
Comment (Pachiru) [inline wiki/Skinning/fr.md]:
```suggestion
**Pour les beatmaps seulement,** si votre skin de beatmap contient un seul élément parmi les ensembles répertoriés dans les critères de classement, il doit contenir tous les autres éléments dudit ensemble. Cela ne s'applique qu'aux skins de beatmap.
```

File: wiki/Skinning/fr.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+## Comment puis-je faire un skin
+
+*Page principale: [Tutoriel de skinning](/wiki/Skinning/Tutorial)*
+
+## Listes d'éléments de skins
+
+La liste des éléments de skinning est divisée en ces catégories:
+
+- [Skinning d'interface](/wiki/Skinning/Interface)
```
Comment (Pachiru) [inline wiki/Skinning/fr.md]:
```suggestion
- [Skinning de l'interface](/wiki/Skinning/Interface)
```

File: wiki/Skinning/fr.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+- [Skinning osu!standard](/wiki/Skinning/osu!)
+- [Skinning osu!taiko](/wiki/Skinning/osu!taiko)
+- [Skinning osu!catch](/wiki/Skinning/osu!catch)
+- [Skinning osu!mania](/wiki/Skinning/osu!mania)
+- [Skinning Sons](/wiki/Skinning/Sounds)
```
Comment (Pachiru) [inline wiki/Skinning/fr.md]:
```suggestion
- [Skinning sonore](/wiki/Skinning/Sounds)
```

File: wiki/Skinning/fr.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+## Voir également
+
+- [skin.ini](/wiki/skin.ini)
+- [Histoire du skinning](/wiki/Skinning/History) - une liste d'anciens éléments de skinning
+- [Compendium](/community/forums/topics/686664) - une liste des skins terminées
+- [FAQ Skinning](/wiki/Skinning/FAQ)
+- [Guides et fils importants](/wiki/Skinning/Guides_and_Important_Threads)
```
Comment (Pachiru) [inline wiki/Skinning/fr.md]:
```suggestion
- [Guides et discussions importants](/wiki/Skinning/Guides_and_Important_Threads)
```

File: wiki/Skinning/fr.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# Skinning
+
+Le skinning est l'une des principales caractéristiques d'osu!. Cela permet aux joueurs de modifier les éléments de skinning originaux pour créer les leurs! Les skins peuvent varier, ils peuvent être amusant, thématisés ou alors être plus professionnel et sobre.
```
Comment (Pachiru) [inline wiki/Skinning/fr.md]:
```suggestion
Le skinning est l'une des principales caractéristiques d'osu!. Cela permet aux joueurs de modifier les éléments de skinning originaux pour créer les leurs! Les skins peuvent varier, ils peuvent être amusants, thématisés ou alors être plus professionnels et sobres.
```
Désolé j'ai totalement oublié les accords!

File: wiki/Skinning/fr.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+**Pour les beatmaps seulement,** si votre skin de beatmap contient un seul élément parmi les ensembles répertoriés dans les critères de classement, il doit contenir tous les autres éléments dudit ensemble. Cela ne s'applique qu'aux skins de beatmap.
+
+## Comment puis-je faire un skin
+
+*Page principale: [Tutoriel de skinning](/wiki/Skinning/Tutorial)*
+
+## Listes d'éléments de skins
+
+La liste des éléments de skinning est divisée en ces catégories:
```
Comment (Pachiru) [inline wiki/Skinning/fr.md]:
```suggestion
La liste des éléments de skinning est divisée en plusieurs catégories:
```

File: wiki/Skinning/fr.md (hunk 6 above)
Comment (Pachiru) [reply wiki/Skinning/fr.md]: Tu as oublié de le modifier.

File: wiki/Skinning/fr.md (hunk 7 above)
Comment (Pachiru) [reply wiki/Skinning/fr.md]: Tu as oublié de le modifier aussi.

File: wiki/Skinning/fr.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+- [Skinning osu!taiko](/wiki/Skinning/osu!taiko)
+- [Skinning osu!catch](/wiki/Skinning/osu!catch)
+- [Skinning osu!mania](/wiki/Skinning/osu!mania)
+- [Skinning sonore](/wiki/Skinning/Sounds)
+
+## Voir également
+
+- [skin.ini](/wiki/skin.ini)
+- [Histoire du skinning](/wiki/Skinning/History) - une liste d'anciens éléments de skinning
+- [Compendium](/community/forums/topics/686664) - une liste des skins terminées
```
Comment (cl8n) [inline wiki/Skinning/fr.md]:
```suggestion
- [Compendium](https://osu.ppy.sh/community/forums/topics/686664) - une liste des skins terminées
```
only remove the hostname from `/wiki` links
could you change this in the other languages as well?

**Pull request merged**